src/Listeners/Ticket/TicketCreatedNotificationListener.php

```diff
@@ -19,13 +19,16 @@ public function handle(NotificationEloquentEventSubscriber $eloquentEventSubscri
             return;
         }
 
+        $notificationRoles = $eloquentEventSubscriber->model
+            ->ticketType
+            ->roles()
+            ->get()
+            ->pluck('id')
+            ->toArray();
         $users = User::query()
-            ->join('model_has_roles', function (JoinClause $join) {
-                $join->on('users.id', '=', 'model_has_roles.model_id')
-                    ->where('model_has_roles.model_type', User::class);
+            ->whereHas('roles', function (Builder $query) use ($notificationRoles) {
```

Comment:
pls change callback to fn (Builder $query) => $query->whereIntegerInRaw('id', $notificationRoles)